GiniSwitchSDK/Example/GiniTariffSDK/SdkBuilder.swift

```diff
@@ -10,9 +10,14 @@ import UIKit
 import GiniSwitchSDK
 
 struct SdkBuilder {
+    
+    static let clientIDKey = "ClientID";
+    static let clientSecretKey = "ClientSecret";
+    static let clientDomainKey = "ClientDomain";
 
     static func customizedSwitchSdk() -> GiniSwitchSdk {
-        let sdk = GiniSwitchSdk(clientId: "TestId", clientSecret: "secret", domain: "gini.net")
+        let credentails = self.clientCredentials()
+        let sdk = GiniSwitchSdk(clientId: credentails.clientId, clientSecret: credentails.clientSecret, domain: credentails.clientDomain)
```

Comment:
It makes easier to review if the line length is shorter. Also if someday you decide to use _SwiftLint_, you will regret to use this length.